README.md

```diff
@@ -377,38 +388,41 @@ Response:
 ```
 
 Response (fault):
-```jsonc
+```
 {
-  "methodname": "StartFetchBlocks",
-  "version": "1.0",
+  "methodname": SetStartingBlock,
+  "version": '1.0',
   "fault": {
-    "string": "Block fetcher already running",
-    "code": "client"
+    "string": 'notFound',
+    "code": 'client'
   },
-  "servicename": "ogmios-datum-cache",
-  "type": "jsonwsp/fault",
+  "servicename": 'ogmios-datum-cache',
+  "type": 'jsonwsp/fault',
```

Comment:
Do we have a reason for single quotes here instead of autoformatter?

Reply:
Not sure what happened here, I'll revert